test/parallel/test-global.js

```diff
@@ -51,6 +51,7 @@ builtinModules.forEach((moduleName) => {
     'setImmediate',
     'setInterval',
     'setTimeout',
+    'structuredClone'
```

Comment:
```suggestion
    'structuredClone',
```
Missing trailing-comma.